src/Element.js

```diff
@@ -243,22 +259,14 @@ export function parent (element) {
  * @return {object}
  */
 export function sibling (element) {
-	return element.uid < Enum.target ? sibling(pick(element)) : element
-}
-
-/**
- * @param {object} element
- * @return {object}
- */
-export function pick (element) {
-	return element.children[0]
+	return element.uid < Enum.target ? sibling(element.children[0]) : element
 }
 
 /**
- * @param {object} element
- * @param {object} value
+ * @param {object?} value
+ * @param {object} props
  * @return {object}
  */
-export function put (element, value) {
-	return element.children[0] = value
+export function resolve (value, props) {
+	return [from(value && 'default' in value ? value.default : value, 0, props), empty()]
```

Comment:
`empty()` at the end feels extraneous (though I'd understand if the logic simplification was worth it)

Reply:
It does simply the logic.